src/main/java/marites/task/Task.java

```diff
@@ -0,0 +1,28 @@
+package marites.task;
+
+public abstract class Task implements java.io.Serializable {
+    private final String description;
+    private boolean isDone;
+    public Task(String description) {
```

Comment:
Consider separating this two statements with an empty line for better readability.